01-CodeReviewEx/Shop-Ex/Program.cs

```diff
@@ -10,11 +10,30 @@ internal class Program
     {
         static void Main(string[] args)
         {
+            Account[] myAccounts = new Account[10];
```

Comment:
jeśli jest to utworzenie tablicy o długości 10 - czy jest potrzebna długość aż 10? wygląda jakby tylko 1 element był wykorzystywany